index.html

```diff
@@ -17,22 +17,177 @@
+                        <li class="g-item1">bootstrap</li>
+                        <li class="g-item1">ruby</li>
+                    </ul>
+                    <button type="button" class="but">See project</button>
+                </div>
+                <div class="grid-item">
+                    <div class="prof">Profesional Art Printing Data</div>
+                    <p class="daily1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="languagess">
+                        <li class="g-item1">html</li>
+                        <li class="g-item1">css</li>
+                        <li class="g-item1">bootstrap</li>
+                        <li class="g-item1">ruby</li>
+                    </ul>
+                    <button type="button" class="but">See project</button>
+                </div>
+            </div>
+        </section>
+       
+        <section>
+            <div class="about">
+                <div>
+                    <h1 class="abouts">About me</h1>
+                    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <button type="button">Get My Resume</button>
+                    <div class="mask"></div>
+                    <img src="./images/Vector 9.png" alt="vector9" class="vector9">
+                </div>
+        
+                    <div class="code">
+                        <div class="ang">
+                            <img src="./images/Rectangle 55.png" alt="triangle" class="cir">
+                            <h2 class="hed">Language</h2>
+                        </div>
+                        <h3>javascript</h3>
+                        <h3>Ruby</h3>
+                        <h3>HTML</h3>
+                        <h3>css</h3>
+                       
+                         <div class="angs">
+                            <img src="./images/Rectangle 56.png" alt="square" class="cir">
+                           <h2 class="hed">Frameworks</h2>
+                         </div>
+                         <h3>Bootstrap</h3>
+                         <h3>Ruby on Rails</h3>
+                         <h3>RSPec</h3>
+                         <h3>CapyBara</h3>
+                         <h3>Selenium</h3>
+                    
+                         <div class="angss">
+                               <img src="./images/Rectangle 57.png" alt="circle" class="cir">
+                                <h2 class="hed">Skills</h2>
+                         </div>
+                         <h3>Codekits</h3>
+                         <h3>Github</h3>
+                         <h3>Codepen</h3>
+                         <h3>Gitlab</h3>
+                         <h3>Terminal</h3>
```

Comment:
:red_circle: As mentioned by the previous reviewer The technology categories ("Languages", "Frameworks" and "Skills") should be in a ul, each of them in a li tag. You can read the [ project requirements ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) :arrow_down: 
![pic](https://user-images.githubusercontent.com/26736582/212466623-eaebe263-61dc-4ac5-9f4c-f78dde2a1fdf.png)